### ISSUE TITLE
Dot is still allowed in validation rule `digits_between`

### ISSUE THREAD

**Issue by taai** [laravel/framework#42540]
### Description:

@taylorotwell [decided](https://github.com/laravel/framework/commit/c113768dbd47de7466d703108eaf8155916d5666) to roll back commits that allowed to use dot (```.```) in validation rule ```digits_between``` (and ```digits```), but forgot to roll back also this commit that started allowing that: https://github.com/laravel/framework/pull/40278

Please, roll back also that commit!

**Comment by driesvints** [comment]
Heya, I sent in a PR for that as well: https://github.com/laravel/framework/pull/42562. Should get in before tomorrow's releases.

Thanks for your commitment to this but I also do hope you understand why I was initially against doing these changes as they could be breaking to users.

**Comment by taai** [comment]
> Thanks for your commitment to this but I also do hope you understand why I was initially against doing these changes as they could be breaking to users.

@driesvints You are doing a huge work reviewing all the issues and PRs - it is understandable that some decisions will be made in a hurry... I don't blame you - the one who wanted a dot to be accepted in the validation rule said that the dot was previously accepted, but he didn't say that "previously" was more than 5 years ago. :man_shrugging: Yes, the resistance from you on this issue was a surprise to me. But when you finally explained that the reason for rejection is possibility of breaking change, I stopped arguing and tried another approach (PR with the third validation rule argument ```strict```). More surprised I was about carelessness of Taylor when he rejected my PR - I guess it is because he is busier than you are. :thinking: After that I turned on the trolling mode and posted my PR with validation rule ```float_between```, because, in my opinion, the situation was ridiculous... No hard feelings?! :wink:
